docs/onboarding/VPN.md

```diff
@@ -13,20 +24,39 @@ Address = X.X.X.X/32
 DNS = 10.20.50.1
 
 [Peer]
-PublicKey = [VPNServerPublicKey]
+PublicKey = Gh7UqjaA2IzrcB9ZNiFco7zVpwHmCeFCHLuny/UETw0=
```

Comment:
is this the actual pubkey for something? Do we wanna make this public?

Reply:
nah, its almost always fine to expose pubkeys :D